packages/cli-repl/test/e2e-direct.spec.ts

```diff
@@ -27,74 +27,76 @@ describe('e2e direct connection', () => {
       });
     });
 
-    it('allows to initialize the replica set', async() => {
-      const replSetConfig = {
-        _id: replSetId,
-        version: 1,
-        members: [
-          { _id: 0, host: await rs0.hostport(), priority: 1 },
-          { _id: 1, host: await rs1.hostport(), priority: 0 },
-          { _id: 2, host: await rs2.hostport(), priority: 0 },
-        ]
-      };
+    context('after rs.initiate()', () => {
```

Comment:
Same as above, these changes are necessary so that the individual test cases below can be run on their own but are otherwise unrelated